### ISSUE TITLE
Time Emb Dim

### ISSUE THREAD

**Issue by angmc** [QData/spacetimeformer#63]
Default Time Emb Dim is 6, however, the paper states that the value used was 12. The example training commands don't specify this hyper parameter. To recreate the results in the paper should 12 be used? Would there be drastically different results due to this hyper parameter? 

Also had to update a line in classification_loss function: `acc = torchmetrics.functional.accuracy(torch.softmax(logits, dim=1),labels,task='multiclass',num_classes=self.d_yc)` in spacetimeformer_model.py. I'm assuming due to versioning differences.
